rotkehlchen/accounting/constants.py

```diff
@@ -35,6 +35,7 @@
         HistoryEventSubType.LIQUIDATE: EventCategory.LIQUIDATION_REWARD,
         HistoryEventSubType.PAYMENT: EventCategory.RECEIVE_PAYMENT,
         HistoryEventSubType.GRANT: EventCategory.RECEIVE_GRANT,
+        HistoryEventSubType.INTEREST: EventCategory.RECEIVE,
```

Comment:
I would create a new EventCategory with its own icon and stuff for this. I am surprised we don't have an interest payment event category for the frontend.